package.json

```diff
@@ -39,7 +39,7 @@
     "**/src/**/*.js": [
       "prettier --write",
       "eslint --fix",
-      "test:staged",
+      "yarn run test:staged",
```

Comment:
i think you should change this to `npm` dont know if everyone is using yarn

Reply:
I see package-lock.json file in the repo. I think changing yarn to npm will be better.